readthedocs/api/v3/proxied_views.py

```diff
@@ -0,0 +1,14 @@
+from readthedocs.core.utils.extend import SettingsOverrideObject
+from readthedocs.embed.v3.views import EmbedAPIBase
+
+
+class ProxiedEmbedAPIBase(EmbedAPIBase):
+
+    # DRF has BasicAuthentication and SessionAuthentication as default classes.
+    # We don't support neither in the community site.
+    authentication_classes = []
```

Comment:
Probably makes sense to make this a Mixin if we're using it on all proxied API's.

Reply:
Good point! I opened an issue to track this https://github.com/readthedocs/readthedocs.org/issues/8542